bindings/dart/lib/src/web/reranker/ai.dart

```diff
@@ -24,7 +25,21 @@ import 'package:xayn_ai_ffi_dart/src/web/worker/message/response.dart'
     show AnalyticsResponse, FaultsResponse, Response, Uint8ListResponse;
 import 'package:xayn_ai_ffi_dart/src/web/worker/oneshot.dart' show Oneshot;
 
+const int kReceiveTimeoutInSec = 15;
```

Comment:
maybe this? similar variable name length and no abbreviation.
```suggestion
const int kReceiveTimeoutSeconds = 15;
```